src/routes/Request/components/RequestActionDialog/RequestActionDialog.js

```diff
@@ -99,6 +100,8 @@ function RequestActionDialog({
     const actionRef = firestore.collection(REQUESTS_ACTIONS_COLLECTION).doc();
     batch.set(actionRef, action);
 
+    analytics.logEvent(`request-action-${actionType}`);
```

Comment:
I like that we are logging this event here - I believe you can pass along extra data for more context, which may be helpful in the future. Seems like passing along requestId and maybe actionId would be helpful?